alpaca/broker/client.py

```diff
@@ -0,0 +1,201 @@
+from typing import List, Optional, Union
+from uuid import UUID
+from pydantic import parse_obj_as
+
+from ..common.enums import BaseURL
+from ..common.rest import RESTClient
+from .models import (
+    Account,
+    AccountCreationRequest,
+    AccountUpdateRequest,
+    CIPInfo,
+    ListAccountsRequest,
+    TradeAccount,
+    BaseActivity,
+    GetAccountActivitiesRequest,
+)
+
+
+def validate_account_id_param(account_id: Union[UUID, str]) -> UUID:
+    """
+    A small helper function to eliminate duplicate checks of account_id parameters to ensure they are
+    valid UUIDs. Upcasts str instances that are valid UUIDs into UUID instances.
+
+    Args:
+        account_id (Union[UUID, str]): The parameter to be validated
+
+    Returns:
+        UUID: The valid UUID instance
+    """
+    # should raise ValueError
+    if type(account_id) == str:
+        account_id = UUID(account_id)
+    elif type(account_id) != UUID:
+        raise ValueError("account_id must be a UUID or a UUID str")
+
+    return account_id
+
+
+class BrokerClient(RESTClient):
+    """
+    Client for accessing Broker API services
+    """
+
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        secret_key: Optional[str] = None,
+        api_version: str = "v1",
+        sandbox: bool = True,
+        raw_data: bool = False,
+    ):
+        """
+        Args:
+            api_key (Optional[str], optional): Broker API key - set sandbox to true if using sandbox keys. Defaults to None.
+            secret_key (Optional[str], optional): Broker API secret key - set sandbox to true if using sandbox keys. Defaults to None.
+            api_version (str, optional): API version. Defaults to 'v1'.
+            sandbox (bool, optional): True if using sandbox mode. Defaults to True.
+            raw_data (bool, optional): True if you want raw response instead of wrapped responses. Defaults to False.
+        """
+        base_url: BaseURL = (
+            BaseURL.BROKER_SANDBOX if sandbox else BaseURL.BROKER_PRODUCTION
+        )
+
+        # TODO: Actually check raw_data everywhere
+        super().__init__(api_key, secret_key, api_version, base_url, sandbox, raw_data)
+
+    # ############################## ACCOUNTS/TRADING ACCOUNTS ################################# #
+
+    def create_account(self, account_data: AccountCreationRequest) -> Account:
+        """
+        Create an account.
+
+        Args:
+            account_data(AccountCreationRequest): The data representing the Account you wish to create
```

Comment:
There should be a space between variable name and variable type
`account_data (AccountCreationRequest): The data representing the Account you wish to create`